assets/panel/panel.js

```diff
@@ -68,10 +63,10 @@ DebuggerPanel.prototype = {
     return { frames, selected };
   },
 
-  destroy: function () {
+  destroy: function() {
```

Comment:
oops, this change was accidental. 

We could update the eslint to be consistent, but we started using prettier on the debugger for the styling and I like the defaults that it has. (even though it's not consistent with devtools in this case)